english/shrooms/engage-man/toothless/enlightenment/eating-marshmellows.md

```diff
@@ -0,0 +1,21 @@
+You're feeling hungry anyway, so a little marshmellow can't hurt. You decide to eat through a good deal so you have plenty of energy.
+
+Feeling enlightened, free, and tripping, the surge of the sugar rush hits as your thoughts begin to drift into the un-speakable.
+You sit down and know somehow you won't be getting up for quite some time. As you trip, memories flash through and alternate universes race past.
+
+You remember all is one organism and that all of life is in the same struggle chasing the same experience of truth, freedom, purity and compassion.
+You realize all of these reactions are really the same experience merely being turned in different ways.
+
+Your eyes close, and a blanket of hallucinated blue spores clouds over you like a blanket.
+You awake in your dream. You are certain you have a mission here of some kind, and race to find it.
+
+You meet Krishna. He informs you that outside is not actually the real world but that you're high in another dimension on another drug
+and that what you currently feel is the 'real reality' is actually a giant trip. This is why you have no recollection of your birth.
+This is why scientists in this dimension can't figure out quantum reality-- reality there is literally a giant paradox of many kinds and
+there is no reality without subjective interpretation by an observer (who happens to be able to change physical particles with his mind and moods)
+-- because the observer himself (you) is dreaming and is the only one that matters in this dream.
+
+Do you:
+[commit these realizations into memory, missing what Krishna says next] (../../../eating-marshmallows.md/steals_from_dreams.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.